jerry-main/main-unix.c

```diff
@@ -109,6 +109,14 @@ jerry_value_is_syntax_error (jerry_value_t error_value) /**< error value */
   }
 
   jerry_size_t err_str_size = jerry_get_string_size (error_name);
+  const char syntax_error_str[] = "SyntaxError";
+
+  if (err_str_size != strlen (syntax_error_str) - 1)
+  {
+    jerry_release_value (error_name);
+    return true;
```

Comment:
`return true;` ? This function is called `jerry_value_is_syntax_error`. This code does the opposite of what's expected.

Reply:
Ohh, that's awkward. I'll fix it.